src/OrchardCore.Modules/OrchardCore.Localization/Manifest.cs

```diff
@@ -9,3 +9,11 @@
     Category = "Internationalization",
     Dependencies = new[] { "OrchardCore.Settings" }
 )]
+#if NET5_0
```

Comment:
OR GREATER ?

Reply:
I tried this before I submit the PR but it doesn't work, and the code is grayed out, I think `OrGreater` is not shipped (frankly I'm not sure)

Reply:
https://github.com/dotnet/designs/blob/main/accepted/2020/or-greater-defines/or-greater-defines.md

Reply:
worst case you can do both NET5_0 and NET5_0_OR_GREATER